src/Microsoft.Data.Sqlite/Strings.resx

```diff
@@ -138,6 +141,9 @@
   <data name="InvalidIsolationLevel" xml:space="preserve">
     <value>The IsolationLevel '{isolationLevel}' is invalid.</value>
   </data>
+  <data name="InvalidIsolationLevelForUnsharedCache" xml:space="preserve">
+    <value>The IsolationLevel '{isolationLevel}' is invalid when the connection is not using Shared Cache mode.</value>
```

Comment:
Also tell how to fix it. E.g. "Specify 'Cache=Shared' in the connection string."